### PR TITLE
Added several new features, adjusted UI

### PR DESCRIPTION
**Pull request by hoff121324**
This pull request contains most of the features from the previous one, but I rewrote it so that it is less obtrusive to existing code.  I also removed the selenium tests and some of the java code that was only used for those tests.  We have some old htmlunit tests that we had used before switching over to selenium, but I decided not to include them in this pull request.

This pull request contains the new configuration menu and the features that are dependent on it, the table filter tools, and the additional table entries for ownership tracking and newly failing tests.

This does not contain the test link urls that are in pull request number 10, but should be able to merge with that pull request easily.

### CODE REVIEW

**Comment by hoff121324** [comment]
I can isolate features such as test filtering, build ownership, and indicators for latest failed builds.  However, this would basically involve basically deleting a bunch of code and then re-adding it as a later commit, which seems silly to me.

File: src/main/java/org/jenkinsci/plugins/testresultsanalyzer/TestResultsAnalyzerAction.java [hunk 1]
```diff
@@ -106,18 +110,32 @@ private JSONArray getBuildsArray(List<Integer> buildList) {
...
-		for (int i = (noOfBuilds - 1); i >= 0; i--) {
-			buildList.add(builds.get(i));
+		for(int i = 0; i < noOfBuilds; i++) {
+			int index = builds.get(i);
+
+			if(showFail || !(compileFailedBuilds.contains(index))) {
+				buildList.add(index);
+			}
 		}
-		Collections.reverse(buildList);
```
Comment (menonvarun) [inline src/main/java/org/jenkinsci/plugins/testresultsanalyzer/TestResultsAnalyzerAction.java]: Please revert this change. As during showing selective builds without this change the build is shown in ascending order rather then descending order.

Comment (hoff121324) [inline src/main/java/org/jenkinsci/plugins/testresultsanalyzer/TestResultsAnalyzerAction.java]: We also changed the order of iteration for the loop before it to reflect this change.  I can revert both if you like, it just seemed a little silly to loop backwards and then reverse the list.

**Comment by hoff121324** [comment]
Some other pull requests have been opened containing some of the features of this one, so I will attempt to split this pull request further down into individual features.
